src/stan-users-guide/odes.Rmd

```diff
@@ -354,25 +354,26 @@ indiscriminately stepping over an important feature of the solution.
 
 ### Tolerance {-}
 
-The relative and absolute tolerance control the accuracy of the
+The relative tolerance RTOL and absolute tolerance ATOL control the accuracy of the
 numerical solution. Specifically, when solving an ODE with unknowns
-$y=(y_1,\dots,y_n)^T$, the
-solver controls estimated local error $e=(e_1,\dots,e_n)^T$ at
-certain step through its weighted root-mean-square norm $\|\cdot\|$
+$y=(y_1,\dots,y_n)^T$, at every step the
+solver controls estimated local error $e=(e_1,\dots,e_n)^T$ through its weighted root-mean-square norm
+(@SerbanHindmarsh:2005, @hairer:1993)
 
 $$
-\|e\| = \sqrt{\sum_{i=1}^n{\frac{e_i^2}{\text{relative\_tolerance}\times y_i + \text{absolute\_tolerance}}}} < 1,
+\|e\| = \sqrt{\sum_{i=1}^n{\frac{1}{n}\frac{e_i^2}{(\text{RTOL}\times y_i + \text{ATOL})^2}}} < 1
```

Comment:
I do not like the combined definition/inequality. Separate these, or just do the inequality.

Reply:
Good point, on it.